app/services/idv/steps/doc_auth_base_step.rb

```diff
@@ -122,6 +122,16 @@ def track_document_state(state)
         doc_auth_log.save!
       end
 
+      # FIXME: bikeshed this name
+      def avoid_fsm?
```

Comment:
I didn't want to repeat `if IdentityConfig.store.in_person_verify_info_controller_enabled` in multiple places, but `avoid_fsm?` may be too succinct? I briefly liked `use_controller?` but I worry that makes it more confusing.

Reply:
```suggestion
      def can_skip_flow_state_machine?
```